sdk/storage/azure-storage-file-datalake/setup.py

```diff
@@ -94,7 +94,7 @@
     ]),
     install_requires=[
         "azure-core<2.0.0,>=1.10.0",
-        "msrest>=0.6.18",
+        "msrest>=0.6.21",
```

Comment:
is there a reason we want to update msrest from 18 to 21?

Reply:
yea autorest now generates and uses a method called `failsafe_deserialize` to deserialize storage errors. This is available >=0.6.21